composer.json

```diff
@@ -17,7 +17,8 @@
     "sonata-project/media-bundle": "^3.0",
     "sonata-project/notification-bundle": "^3.1",
     "symfony/framework-bundle": "^3.3 || ^4.0",
-    "tinify/tinify": "^1.5"
+    "tinify/tinify": "^1.5",
+    "zendframework/zenddiagnostics": "^1.5.0"
```

Comment:
We should probably suggest `liip/monitor-bundle` because of the required tag on the service definition